### PR TITLE
Add color option to milestone card

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
Add color option to milestone card

### Solution:
required changes done

## WtId:
Dev606

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
go to milestone page  and click on add new milestone or click on edit milestone

### CODE REVIEW

File: src/app/body/milestone-details/edit-milestone/edit-milestone.component.css [hunk 1]
```diff
@@ -97,3 +97,30 @@ input:focus {
     color: var(--secondary-color);
     font-size: 48px;
 }
+
+:host {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
```
Comment (viv9k) [inline src/app/body/milestone-details/edit-milestone/edit-milestone.component.css:105]: This is not required

File: src/app/body/milestone/add-milestone/add-milestone.component.css [hunk 2]
```diff
@@ -70,4 +70,31 @@ input:focus {
 #SuccessIcon{
     color: var(--secondary-color);
     font-size: 55px;
+}
+
+:host {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
```
Comment (viv9k) [inline src/app/body/milestone/add-milestone/add-milestone.component.css:79]: This is not required

**Review by viv9k — CHANGES_REQUESTED** [review]
Need demo and fix required things

File: src/app/body/milestone-details/milestone-details.component.html [hunk 3]
```diff
@@ -51,11 +51,12 @@
       <div class="col-md-4">
         <div class="card p-3">
           <div class="row py-3">
-            <div class="col text-center">
+            <div class="col-11 text-center">
               <span class="title">
                 {{milestoneData.Title}}
               </span>
             </div>
+            <div id="milColor" [ngStyle]="{'background-color': color}"></div>
```
Comment (viv9k) [inline src/app/body/milestone-details/milestone-details.component.html:59]: Give this div class of col

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the fix

File: src/app/body/milestone-details/milestone-details.component.html (hunk 3 above)
Comment (Bankar-Swapnil) [reply src/app/body/milestone-details/milestone-details.component.html:59]: sir class col to this div disturb ui of card without that class it works fine and responsive too

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**